src/locale/locales/ja/messages.po

```diff
@@ -5089,9 +5043,9 @@ msgstr "サポートフォームは移動しました。サポートが必要な
 msgid "The Terms of Service have been moved to"
 msgstr "サービス規約は移動しました"
 
-#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:141
-msgid "There are many feeds to try:"
-msgstr "試せるフィードはたくさんあります："
+#: src/screens/Settings/components/DeactivateAccountDialog.tsx:35
+msgid "There is no time limit for account deactivation, come back any time."
+msgstr "アカウントの無効化に期限はありません。いつでも戻ってこれます。"
```

Comment:
```suggestion
msgstr "アカウントの無効化に期限はありません。いつでも戻ってこられます。"
```

Reply:
fixed in https://github.com/bluesky-social/social-app/pull/4477/commits/fa74fb9dfef6bdeeaf24c589ac52b0f221c51319 . thanks.